data/golden/yso.csv

```diff
@@ -0,0 +1,101 @@
+ra,dec
```

Comment:
The RA/Dec plot will be like a single blob. Do we want to get more generic?

Reply:
We will add objects from other parts later.